scripts/setup.js

```diff
@@ -22,59 +26,67 @@ async function main() {
 
   const ethSender = await ethers.getSigner(accounts[0].address);
   // Send 0.5 ETH to all addresses
-  for (let i = 0; i < 50; i++) {
-    await ethSender.sendTransaction({
-      to: keyPairs.addresses[i],
-      value: ethers.utils.parseEther('0.5')
-    });
-  }
+  // for (let i = 0; i < 50; i++) {
+  await ethSender.sendTransaction({
+    to: '0x8028Ef7ADA45AA7fa31EdaE7d6C30BfA5fb3cf0B',
+    value: ethers.utils.parseEther('5')
+  });
+  // }
```

Comment:
Looks like this updates the script to only fund one wallet.

If that's the case, should we remove the old comment and commented out lines?

Reply:
Good call, I commented these out early on because I was having some issues with accounts being funded. I'll double check if they can be re-enabled in the interest of keeping the file as close to the original as possible (same as comment below)

Reply:
I reverted this back to what it was and tested that its working properly. I updated some of the logs to be more specific.